src/bandwidthRtc.ts

```diff
@@ -38,6 +42,12 @@ class BandwidthRtc {
     this.setCameraEnabled = this.setCameraEnabled.bind(this);
   }
 
+  public sendDtmf(tone: string) {
```

Comment:
If we move to using a DtmfSender per stream, we would also need to take a `streamId?: string` parameter here to allow the developer to specify which stream to send the tone on. Otherwise, I think it sends it on all streams by default?